content/_index.html

```diff
@@ -168,7 +168,7 @@ <h5 class="card-title">How To Get Involved</h5>
         <div class="card">
           <div class="card-body">
             <h5 class="card-title">How To Get Support</h5>
-            <p class="card-text">Found an issue or have a problem? Community support happens in the #rancher-desktop channel in Slack and in the issues queue, where you can search and file new issues.</p>
+            <p class="card-text">Found an issue or have a problem? Community support happens in the <b>#rancher-desktop</b> channel in Slack and in github issue, where you can search and file new issues.</p>
```

Comment:
GitHub Issues is the name of the product, so I think we should treat it like a proper noun. What do you think?

```suggestion
            <p class="card-text">Found an issue or have a problem? Community support happens in the <b>#rancher-desktop</b> channel in Slack and in GitHub Issues, where you can search and file new issues.</p>
```